src/arch/ucx/machine.C

```diff
@@ -279,9 +280,9 @@ void LrtsInit(int *argc, char ***argv, int *numNodes, int *myNodeID)
     }
 
     // Any RNDV thresh is allowed
-    if (!CmiGetArgInt(*argv, "+ucx_rndv_thresh", &ucxCtx.eagerSize)) {
-        ucxCtx.eagerSize = UCX_MSG_PROBE_THRESH;
-    }
+    int thresh = 0;
+    CmiGetArgInt(*argv, "+ucx_rndv_thresh", &ucxCtx.eagerSize);
+    ucxCtx.eagerSize = std::max(UCX_MSG_PROBE_THRESH, thresh);
```

Comment:
This can't be right - `thresh` is never re-assigned from its initial `0`, the command line argument is now entirely ignored, and `ucxCtx.eagerSize` is always set to 32768.

Reply:
```
  std::max(UCX_MSG_PROBE_THRESH, thresh);
= std::max(32768, 0);
= 32768
```

Reply:
yes, you are right. Would need to `CmiGetArgInt(*argv, "+ucx_rndv_thresh", &thresh);`
Waiting for a an answer on maximal size of `NcpyOperationInfo` to implement clean patch. Will apply this comment as well then